docs/Taps.md

```diff
@@ -16,12 +16,14 @@ mistydemeo/tigerbrew
 dunn/emacs
 ```
 
+<!-- vale Homebrew.Terms = OFF -->
```

Comment:
What's this doing?

Reply:
Filtering out the one "repo" we want to keep as "repo" and not force a change to "repository". It's like the Rubocop disable lines.

Reply:
Cool, would be nice to document each of these cases as to why it's being disabled?